src/content/docs/browser/new-relic-browser/browser-apis/setcustomattribute.mdx

```diff
@@ -72,17 +80,33 @@ Errors for custom attributes will be included in events on the [JS Errors page](
 
     <tr>
       <td>
-        `$value`
+        `value`
 
-        _string_ OR _integer_
+        _string_ OR _integer_ OR _null_
       </td>
 
       <td>
         Required. Value of the attribute. Appears as the value in the named attribute column in the `PageView` event. It will appear as a column in the `PageAction` event if you are using it. Custom attribute values cannot be complex objects, only simple types such as Strings and Integers.
 
+        Passing a `null` value will act as an un-setting of any existing attribute by the same name.
```

Comment:
@cwli24, could we simplify this sentence like this: "Passing a `null` value unsets any existing attributes by the same name."?